internal/core/domain/publicKey.go

```diff
@@ -0,0 +1,50 @@
+package domain
+
+import (
+	"bytes"
+
+	"github.com/iden3/go-iden3-crypto/babyjub"
+)
+
+const (
+	bjjAuthSchemaURL = "https://schema.iden3.io/core/json/auth.json"
```

Comment:
Use constant from library.
I'll add to https://github.com/iden3/go-schema-processor/blob/main/verifiable/constants.go